src/SFA.DAS.ApprenticeFeedback.Jobs/Infrastructure/Api/IApprenticeFeedbackEmailTransactionApi.cs

```diff
@@ -0,0 +1,14 @@
+﻿
+using RestEase;
+using SFA.DAS.ApprenticeFeedback.Jobs.Infrastructure.Api.Requests;
+using System.Threading.Tasks;
+
+
+namespace SFA.DAS.ApprenticeCommitments.Jobs.Api
```

Comment:
Should be able to remove this and add in the code into the singular Outer Api Interface here: https://github.com/SkillsFundingAgency/das-apprentice-feedback-jobs/blob/main/src/SFA.DAS.ApprenticeFeedback.Jobs/Infrastructure/Api/IApprenticeFeedbackApi.cs

Reply:
done